src/libcore/num/mod.rs

```diff
@@ -2508,11 +2508,11 @@ macro_rules! try_from_unbounded {
     ($source:ty, $($target:ty),*) => {$(
         #[unstable(feature = "try_from", issue = "33417")]
         impl TryFrom<$source> for $target {
-            type Error = TryFromIntError;
+            type Error = Infallible;
```

Comment:
Hmm, this might need to remain `TryFromIntError` to avoid portability hazards...

Reply:
Isn't this macro only being called when the target's pointer width makes the conversion infallible?

Reply:
Is this macro used anymore?

Reply:
Yes, it's used in the target-specific `ptr_try_from_impls` module.

Reply:
I don't think we want these impls changing based on your target.

Reply:
Most calls to this macro were removed in this PR. The ones that are still there are the conversions from specifically sized integer types to `usize`/`isize`. For example, `u32` is only an infallible conversion to `usize` if the target's pointer width is 32 or higher.